test/factories.rb

```diff
@@ -0,0 +1,50 @@
+FactoryGirl.define do
+  factory :product do
+    name "My Amazing Article"
+    available true
+    on_hand_count 10
+    price 10.00
+  end
+
+  factory :user do
+    user_name 'John Doe'
```

Comment:
user_name like the login username (fiestacasey) or like the user's name (Casey Lopez)?